### PR TITLE
adding messagefeedservlet hardcode

### PR DESCRIPTION
**Pull request by conniewu1**
-Added messagefeedservlet class 
-added a method to and refactored datastore

### CODE REVIEW

**Comment by conniewu1** [comment]
changed code in messagefeedservlet class

File: src/main/webapp/js/feed-loader.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+// Fetch messages and add them to the page.
+function fetchMessages(){
+    const url = '/feed';
+    fetch(url).then((response) => {
+        return response.json();
+}).then((messages) => {
```
Comment (matthew-chinn) [inline src/main/webapp/js/feed-loader.js]: Promises can make code hard to read if we don't keep it styled nicely :( At our next meeting, we can look into adding an automated formatter to style our guide for us, but for now, can we format it so it looks something like this:
```
fetch(url)
    .then((response) => {
           <code goes here>
    }).then((messages) => {
          <code goes here>
    });
```

**Review by matthew-chinn — CHANGES_REQUESTED** [review]

File: src/main/webapp/js/feed-loader.js (hunk 1 above)
Comment (conniewu1) [reply src/main/webapp/js/feed-loader.js]: got it, thanks!!

**Review by matthew-chinn — APPROVED** [review]
Thanks Connie!!

**Pull request merged**